src/client.h

```diff
@@ -286,6 +286,8 @@ class CClient : public QObject
         Channel.GetBufErrorRates ( vecErrRates, dLimit, dMaxUpLimit );
     }
 
+    CProtocol* getConnLessProtocol() { return &ConnLessProtocol; }
```

Comment:
Hm.  `pClient` gets passed to `ClientRpc`, which isn't a problem for threading.  That should make calls to `pClient->anything()` thread-safe, right?

I'd just be more comfortable not to dig around inside pClient to the internals -- if pClient should expose a change, it should have its own slot exposing that and signal it itself.

Reply:
@riban-bw

Reply:
@riban-bw @ann0see -- OK, can we add a "TODO" here to fix this in a future refactoring, noting it's been agreed as out of scope for this PR?

Reply:
Just this to go for me.

Reply:
Fixed.